sdk/data/azappconfig/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "go",
   "TagPrefix": "go/data/azappconfig",
-  "Tag": "go/data/azappconfig_560d04663e"
+  "Tag": "go/data/azappconfig_88f23302b7"
```

Comment:
How did this change recordings?

Reply:
Previously, we were [always adding back the body](https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/data/azappconfig/internal/auth/policy_hmac_auth.go#L59) even if it was `nil`.